recipes/anaconda-cloud-auth/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "anaconda-cloud-auth" %}
-{% set version = "0.1.0a7" %}
+{% set version = "0.1.0a8" %}
```

Comment:
This one falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
Thank you for that reference. We can bump to a proper release, instead of using pre-releases. I wasn't aware of this consideration. I will work to get those released, and then I will refactor and ping.

Reply:
A stable release is definitely better here.

Reply:
@ocefpaf I've just bumped to our first stable minor release for all the packages.